tests/ui/parser/require-parens-for-chained-comparison.rs

```diff
@@ -24,12 +24,14 @@ fn main() {
     //~| HELP use `::<...>` instead of `<...>` to specify lifetime, type, or const arguments
     //~| ERROR expected
     //~| HELP add `'` to close the char literal
+    //~| ERROR invalid label name
 
     f<'_>();
     //~^ comparison operators cannot be chained
     //~| HELP use `::<...>` instead of `<...>` to specify lifetime, type, or const arguments
     //~| ERROR expected
     //~| HELP add `'` to close the char literal
+    //~| ERROR invalid label name
```

Comment:
I find it slightly unexpected that the error is about labels instead of about lifetimes. I guess this is because it is not even trying to parse this as a path at this point.

Reply:
Yes, this is being parsed as "`f` is less than..."